README.md

```diff
@@ -23,6 +23,13 @@ To set up `wrangler` to work with your Cloudflare account, use the following com
 
     ⚠️ NEVER PUBLISH CREDENTIALS TO VERSION CONTROL! ⚠️
 
+##
```

Comment:
so- i think we want to be careful here because not all users will have this issue and not all users are on a Mac, with Homebrew.

i'd consider taking step 1 of the quickstart out and making it its own "Installation" section. then i would add this as a note saying if you get an SSL error and are on a mac with homebrew (link to how to install homebrew) and then give the commend :)

Reply:
Good point; fixed!